pringles/backends/statics/basic.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html class="basic-page">
+
+<head>
+  <meta charset="UTF-8">
+  <title>DEVS Diagrammer</title>
+  <meta name="viewport" content="width=device-width, user-scalable=no">
+
+  <link rel="stylesheet" href="{{ url_prefix }}/_static/css/style.css" />
+
+  <script src="{{ url_prefix }}/_static/vendors/js/createjs/createjs-2013.12.12.min.js"></script>
+  <script src="{{ url_prefix }}/_static/vendors/js/jquery-3.2.1/jquery-3.2.1.min.js"></script>
+  <script src="{{ url_prefix }}/_static/js/ColorCodes.js"></script>
+  <script src="{{ url_prefix }}/_static/js/main.js"></script>
+  <script src="{{ url_prefix }}/_static/js/manifest.js"></script>
+  <script src="{{ url_prefix }}/_static/js/toposort.js"></script>
+  <script src="{{ url_prefix }}/_static/js/Square.js"></script>
+  <script src="{{ url_prefix }}/_static/js/Port.js"></script>
+  <script src="{{ url_prefix }}/_static/js/Link.js"></script>
+  <script src="{{ url_prefix }}/_static/js/JSONModelGraphics.js"></script>
+  <script src="{{ url_prefix }}/_static/js/Model.js"></script>
+  <script src="{{ url_prefix }}/_static/js/Canvas.js"></script>
+
+  <script type="text/javascript">
+    modelAsJson = '{% raw model_source %}'
+    $(document).ready(function () {
+      console.info("About to start drawing model");
+      menu_init();
+      parse_query_params(modelAsJson);
+    });
+  </script>
+
+</head>
+
+<body>
+
+  <!-- hiden button for export JSON -->
+  <a id="download" style="display:none"></a>
+  <!-- hiden button for export JSON -->
+
+  <div id="devs_diagrammer">
+    <div id='body-wrapper'>
+      <div id="models-wrapper"></div>
+      <!-- <div id="menu-wrapper">
```

Comment:
why delete the buttons?

Reply:
The whole UI is messed up when inserting it into jupyter. For now, I'll leave them this way. 
Once you check the whole CSS config, you can uncomment them.